lib/Tool/Requirements.php

```diff
@@ -615,6 +616,16 @@ public static function checkPhp()
             'state' => class_exists('Imagick') ? Check::STATE_OK : Check::STATE_WARNING,
         ]);
 
+        if(class_exists('Imagick')) {
+            $imageMagickLcmsDelegateInstalledProcess = Process::fromShellCommandline('convert -list configure | grep DELEGATES');
+            $imageMagickLcmsDelegateInstalledProcess->run();
+            $checks[] = new Check([
+                'name' => 'ImageMagick LCMS delegate',
+                'link' => 'https://talk.pimcore.org/t/cmyk-to-rgb-displaying-as-negative/908/17',
```

Comment:
I'd add a note about LCMS here: https://github.com/pimcore/pimcore/blob/7fff27a970182dc18717b3aa1727d29e388ba72d/doc/Development_Documentation/23_Installation_and_Upgrade/01_System_Requirements.md#L40 and change the link to the docs.

Reply:
Done.